### ISSUE TITLE
Use markers to indicate exit_message in waterfall plot

### ISSUE THREAD

**Issue by FFroehlich** [ICB-DCM/pyPESTO#66]
Would be nice to have this as quick visual inspection

**Comment by yannikschaelte** [comment]
what precisely do you mean? the exit status somehow plottable on the graph (i.e. whether converged, or out of function evaluations)?

**Comment by JanHasenauer** [comment]
yes, this would be a nice feature. maybe coloring of points by a exit flag. alternatively the shape might be changed.

**Comment by FFroehlich** [comment]
https://github.com/ICB-DCM/pyPESTO/blob/808fe96e351864bbed4c977012c2305d029058e8/pypesto/optimize/optimizer.py#L87

I want a way to visualize the message field in OptimizerResult. Currently the clustering information is used for coloring, so marker is probably the thing to go for (could have options to specify what goes where)
